src/gui/map/map_editor.cpp

```diff
@@ -905,9 +905,9 @@ void MapEditorController::createActions()
 	open_template_act = newAction("opentemplate", tr("Open template..."), this, SLOT(openTemplateClicked()), nullptr, QString{}, "templates_menu.html");
 	reopen_template_act = newAction("reopentemplate", tr("Reopen template..."), this, SLOT(reopenTemplateClicked()), nullptr, QString{}, "templates_menu.html");
 	
-	tags_window_act = newCheckAction("tagswindow", tr("Tag editor"), this, SLOT(showTagsWindow(bool)), "window-new", tr("Show/Hide the tag editor window"), "tag_editor.html");
+	tags_window_act = newCheckAction("tagswindow", tr("Tag Editor"), this, SLOT(showTagsWindow(bool)), "window-new", tr("Show/Hide the tag editor window"), "tag_editor.html");
 
-	tag_select_window_act = newCheckAction("tagselectwindow", tr("Tag Selection"), this, SLOT(showTagSelectWindow(bool)), "tag-selector", tr("Show/Hide the tag selection window"), "tag_selector.html");
+	tag_select_window_act = newCheckAction("tagselectwindow", tr("Tag Selector"), this, SLOT(showTagSelectWindow(bool)), "tag-selector", tr("Show/Hide the tag selector window"), "tag_selector.html");
```

Comment:
This has the wrong capitalization, given its context.

Apart from capitalization, I'm reluctant to accept this change because a) we would need to update (source strings in) translations at the same time, b) I still think the "Tag selector" is better made an extra feature of the new (v0.8) "Find objects" dialog, and c) I have no clue what native speakers (English) consider a better match.

Reply:
Are there similar tools in JOSM, Merkaartor, QGIS? Maybe, this feature in those GIS already has some better & more correct name.

Reply:
> I have no clue what native speakers (English) consider a better match.

Please, ask it on "StackExchange/GIS"
* https://gis.stackexchange.com/

Reply:
Again, my proposal is to remove this menu item and the dock widget, and make it a feature of the "Find objects" dialog. It would call it query editor there. It is almost finished, because I wanted to do this in v0.8.0:

![query-editor](https://user-images.githubusercontent.com/13567791/33228050-72d94d26-d1b2-11e7-9dce-fa0e9dfb7cdb.png)

Reply:
> my proposal is to remove this menu item and the dock widget, and make it a feature of the "Find objects" dialog. It would call it query editor there.

OK, I agree then.

But, maybe, also rename "Tag Editor" to "Tag window".

P.S.: Or much better use "widget" instead of "window": "Colors widget", "Symbols widget", "Tags widget", etc.